docker/readme.md

```diff
@@ -0,0 +1,27 @@
+# Building SeaHorn with Docker and running tests
+
+## To build SeaHorn with Docker:
+```shell
+docker build --build-arg UBUNTU=xenial --build-arg BUILD_TYPE=Release -t seahorn_xenial_rel -f seahorn-full-size-rel.Dockerfile .
+docker run -v `pwd`:/host -it seahorn_xenial_rel
+```
+This will automatically download all dependancies from a base image (generated by `seahorn-build.Dockerfile`) and build SeaHorn under `/seahorn/build`.
+
+SeaHorn's install directory is added to PATH, and a tarball with it is ready under the current working directory (on the host).
+
+The possible build arguments are:
+- UBUNTU: trusty, xenial
+- BUILD_TYPE: Release, Debug
+
+NOTE: Debug build type is not supported at the moment.
```

Comment:
If this comes from some of my previous files, this should not longer be a case -- we have dependencies for [Debug, Release] x [trusty, xenial] hosted in seahorn-deps.